pkg/ensurance/collector/nodelocal/cpu.go

```diff
@@ -13,13 +16,13 @@ import (
 )
 
 const (
-	cpuCollectorName = "cpu"
-	MAX_PERCENTAGE   = 100
-	MIN_PERCENTAGE   = 0
+	cpuCollectorName     = "cpu"
+	cpuLoadCollectorName = "cpuLoad"
 )
 
 func init() {
-	registerMetrics(cpuCollectorName, []types.MetricName{types.MetricNameCpuTotalUsage, types.MetricNameCpuTotalUtilization}, NewCPUCollector)
+	registerMetrics(cpuCollectorName, []types.MetricName{types.MetricNameCpuTotalUsage, types.MetricNameCpuTotalUtilization}, collectCPU)
```

Comment:
registerCollector?
metrics generally means promethus collector

Reply:
I modified it.